modules/web/src/com/haulmont/cuba/web/toolkit/ui/CubaGrid.java

```diff
@@ -284,11 +264,30 @@ protected boolean isEditorFieldsValid() {
         }
     }
 
-    public boolean isCrossFieldValidate() {
+    protected void itemCrossFieldValidate() throws Validator.InvalidValueException {
```

Comment:
Do not through an exception from this method. See the discussion above for more details.